src/store/actions/RelatedWordsActions.js

```diff
@@ -0,0 +1,36 @@
+import * as actionType from "./actionTypes"
+import {getRelatedWordsAPI} from "../../utilities/apiHandler"
+
+export const getRelatedWords = (payload) => {
```

Comment:
We should later align some words 
i.e. RelatedWords => neighbours (align with backend, frontend)